src/devices/machine/mm5740.h

```diff
@@ -0,0 +1,210 @@
+				 TRI-STATE outputs with direct TTL 
+				 compatibility. When the output enable
+				 input (Pin 15) is high, these outputs
+				 are in the third state.
+
+Data Strobe Output   13          The function of this pin is to indicate
+				 that valid data has been entered by the
+				 keyboard and is ready for acceptance. 
+				 An active data strobe may be operated
+				 in the pulse or level mode as indicated
+				 by the timing diagram.
+
+Data Strobe Control  14          The basic purpose of this input is to
+				 provde data strobe output pulse width
+				 control. When connected to the data
+				 strobe output (Pin 13), the data
+				 strobe will exhibit a one bit wide
+				 pulse width. The pulse width may be
+				 varied by interposing an RC network
+				 between the data strobe output and
+				 the strobe control input. For level
+				 mode of operation the data strobe
+				 control input may be tied to Vss
+				 or to the data strobe output.
+
+Output Enable        15          This input serves to TRI-STATE the
+				 data output(B1-B9) lines. In addition,
+				 it control the return of the data
+				 strobe to the idle condition (low
+				 state)which is needed in the level
+				 strobe mode of operation.
+
+Repeat               16          The repeat input is designed to
+				 accept a repeat signal via the repeat
+				 key. One data strobe will be issued
+				 for each positive interval of the
+				 repeat signal. Thus, if a 10 Hz
+				 signal is applied to the repeat
+				 switch, a 10 character per second
+				 data strobe will be issued when
+				 a data key and the repeat key are
+				 held depressed.
+
+Key-Bounce Mask      17          This pin is intended as a timing
+				 node to mas switch key-bounce. The
+				 mask time interval is generated by
+				 connecting a capacitor to this pin.
+
+Shift                21          When this input is brought to a
+				 logic "0" (Vss) level, the encoder
+				 will assume the shifted character mode.
+
+Control              19          A logic "0" places the encoder in
+				 the control character mode.
+
+Shift Lock I/O       20          This pin is intended to serve as an
+				 input when the shift lock key is
+				 depressed. It places the encoder
+				 in the shift mode. Upon release of
+				 the key, the shift mode will be
+				 maintained and this pin will serve
+				 as an output to drive an indicator.
+				 This function is reset by depressing
+				 the shift key.
+
+Clock                3           A TTL compatible clock signal is
+				 applied to this pin. A bit time is
+				 defined as the time from one
+				 negative going transition to the
+				 succeeding negative going transition
+				 of the clock.
+
+Vss                  32          +5.0V supply
+
+Vll                  2           Ground
+
+Vgg                  18          -12V supply
+
+
+**********************************************************************/
+
+#pragma once
+
+#ifndef __MM5740__
+#define __MM5740__
```

Comment:
It seems really common for the headers in this project to use double underscores. And many of the ones that have a single underscore are followed by a capital letter, which is also reserved.